Source/WebKit/UIProcess/Automation/protocol/BidiScript.json

```
@@ -0,0 +1,174 @@
+            "description": "Unique identifier for a script realm. Defined at <https://tc39.es/ecma262/#realm>"
+        },
+        {
+            "id": "RemoteValue",
+            "type": "object",
+            "properties": [
+                { "name": "type", "$ref": "BidiScript.RemoteValueType" },
+                { "name": "value", "type": "any", "optional": true },
+                { "name": "handle", "$ref": "BidiScript.Handle", "optional": true, "description": "Represents a reference to a remote Node. Applicable if `type` property is `node`, `nodelist`, `htmlcollection`." },
+                { "name": "internalId", "$ref": "BidiScript.InternalId", "optional": true, "description": "Represents a reference to a remote script value. If both `handle` and `internalId` are specified, `internalId` takes precedence." }
+            ]
+        },
+        {
+            "id": "RemoteValueType",
+            "type": "string",
+            "enum": [
+                "array",
+                "arraybuffer",
+                "date",
+                "error",
+                "function",
+                "htmlcollection",
+                "map",
+                "node",
+                "nodelist",
+                "object",
+                "promise",
+                "proxy",
+                "set",
+                "symbol",
+                "typedarray",
+                "weakmap",
+                "weakset",
+                "window"
+            ]
+        },
+        {
+            "id": "SerializationOptions",
+            "type": "object",
+            "properties": [
+                { "name": "maxDomDepth", "type": "number", "optional": true },
+                { "name": "maxObjectDepth", "type": "number", "optional": true },
+                { "name": "includesShadowTree", "type": "string", "enum": ["none", "open", "all"], "optional": true}
+            ]
+        },
+        {
+            "id": "StackTrace",
+            "type": "object",
+            "properties": [
+                { "name": "callFrames", "type": "array", "items": { "$ref": "BidiScript.StackFrame" } }
+            ]
+        },
+        {
+            "id": "StackFrame",
+            "type": "object",
+            "properties": [
+                { "name": "lineNumber", "type": "number" },
+                { "name": "columnNumber", "type": "number" },
+                { "name": "functionName", "type": "string" },
+                { "name": "url", "type": "string" }
+            ]
+        },
+        {
+            "id": "Target",
+            "type": "object",
+            "description": "A type that represents a place to evaluate JavaScript. It is either a BrowsingContext or a Realm.",
+            "properties": [
+                { "name": "realm", "$ref": "BidiScript.Realm", "optional": true },
+                { "name": "context", "$ref": "BidiBrowsingContext.BrowsingContext", "optional": true },
+                { "name": "value", "type": "any" }
```

What's this `"value"` key? Shouldn't it be the optional `sandbox` for `script.ContextTarget` specialization?

https://w3c.github.io/webdriver-bidi/#type-script-Target

```suggestion
                { "name": "sandbox", "type": "string", "optional": true }
```

---

Uh, no idea! Probably added to wrong command.